app/pages/project/affinity/AntiAffinityGroupPage.tsx

```diff
@@ -84,15 +89,71 @@ export default function AntiAffinityPage() {
     memberList({ antiAffinityGroup, project }).optionsFn()
   )
   const membersCount = members.items.length
-  const columns = useMemo(
-    () => [
+
+  const { mutateAsync: removeMember } = useApiMutation(
+    'antiAffinityGroupMemberInstanceDelete',
+    {
+      onSuccess(_data, variables) {
+        queryClient.invalidateEndpoint('antiAffinityGroupMemberList')
+        queryClient.invalidateEndpoint('antiAffinityGroupView')
+        addToast(<>Member <HL>{variables.path.instance}</HL> removed from anti-affinity group <HL>{group.name}</HL></>) // prettier-ignore
+      },
+    }
+  )
+
+  const makeActions = useCallback(
+    (antiAffinityGroupMember: AntiAffinityGroupMember): MenuAction[] => [
+      {
+        label: 'Copy instance ID',
+        onActivate() {
+          navigator.clipboard.writeText(antiAffinityGroupMember.value.id)
+          addToast('ID copied to clipboard')
```

Comment:
We don't do the toast on any of the other copy ID row actions, as far as I can tell. I think you're right that it would be better to have some feedback, but the toast is not ideal because it's so far away. See #2416